runtime/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -193,6 +193,7 @@ public class JsonBuilder {
     public var indent: String = "    "
     public var useArrayPolymorphism: Boolean = false
     public var classDiscriminator: String = "type"
+    public var supportAlternativeNames: Boolean = false
```

Comment:
Please consider naming it "useAlternativeNames" to have a shorter option name.
It's supported, users just have to enable/use it :)